autospotting.go

```diff
@@ -98,6 +105,9 @@ func (c *cfgData) parseCommandLineFlags() {
 			autospotting.OnDemandPercentageLong+
 			"\n\tIt is ignored if min_on_demand_number is also set.")
 
+	flag.BoolVar(&c.KeepInstanceType, "keep_instance_type", false,
+		"If specified the instance type of the on-demains instances "+
+			"is matched")
```

Comment:
I'd use "If specified, the spot instances will have the same instance type as the initial on-demand instances", I think it's a bit more clear